ingest/config/optional.yaml

```diff
@@ -10,8 +10,16 @@ upload:
     files_to_upload:
       genbank.ndjson.xz: data/genbank.ndjson
       all_sequences.ndjson.xz: data/sequences.ndjson
-      metadata.tsv.gz: results/metadata.tsv
-      sequences.fasta.xz: results/sequences.fasta
+      metadata_all.tsv.zst: results/metadata_all.tsv
+      sequences_all.fasta.zst: results/sequences_all.fasta
+      metadata_denv1.tsv.zst: results/metadata_denv1.tsv
+      sequences_denv1.fasta.zst: results/sequences_denv1.fasta
+      metadata_denv2.tsv.zst: results/metadata_denv2.tsv
+      sequences_denv2.fasta.zst: results/sequences_denv2.fasta
+      metadata_denv3.tsv.zst: results/metadata_denv3.tsv
+      sequences_denv3.fasta.zst: results/sequences_denv3.fasta
+      metadata_denv4.tsv.zst: results/metadata_denv4.tsv
+      sequences_denv4.fasta.zst: results/sequences_denv4.fasta
       alignment.fasta.xz: data/alignment.fasta
       insertions.csv.gz: data/insertions.csv
       translations.zip: data/translations.zip
```

Comment:
Thanks for flagging! I opted for cleaning up these files since we are not using them yet: 6dbc1afe55cf080f8bb450d7ebc2b485be878e08

However, if we do move toward cached nextclade runs, we can add the files (e.g. a pair of alignment and translation files for each serotype) at that time.

Reply:
Thanks for cleaning those up! 

FWIW, these files are not just used for the cached Nextclade runs. It's mostly to provide them as public files that others can use for downstream analysis.

Reply:
Oh, good to know and makes sense! :D